tests/pmap_test.py

```diff
@@ -473,6 +474,22 @@ def testVmapOfPmap(self):
     bx = vmap(f1)(ax)
     self.assertAllClose(ax, bx, check_dtypes=False)
 
+  def testVmapOfPmap2(self):
+    N_DEVICES = xla_bridge.device_count()
+    keys = random.split(random.PRNGKey(1), 13)  # [13, 2]
+
+    @pmap
+    def g(key):
+      params = random.normal(key, ())
+      return 0.
+
+    @vmap
+    def s(keys):
+      keys = np.broadcast_to(keys, (N_DEVICES,) + keys.shape)
+      return g(keys)
+
+    s(keys)
```

Comment:
Maybe validate the contents of the output, e.g. shape of the array?

Reply:
Great idea.

Reply:
Done.